### DISCUSSION TITLE
Add M1 (Apple silicon or arm64-apple-darwin) build of Julia

### DISCUSSION THREAD

**Discussion by synkathairo** [Homebrew/discussions#2691] [Writing Formulae/Casks]
https://julialang.org/downloads/ the M1 build of Julia is live on the Julia website per version 1.7.1

Would it be possible to update https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/julia.rb the Julia formula to reflect this change?

**Comment by synkathairo** [comment]
I wanted to file a pull request regarding this update but wasn't sure how the system of bottle binaries and shasums etc works

**Comment by carlocab** [comment]
If you're able to build Julia from source on M1 after removing this line: https://github.com/Homebrew/homebrew-core/blob/a71754b9fa98487ffab5dc0b8a4ec95ba0e1cc3b/Formula/julia.rb#L18

then sure. But I don't think it's possible. See https://github.com/Homebrew/homebrew-core/blob/a71754b9fa98487ffab5dc0b8a4ec95ba0e1cc3b/Formula/julia.rb#L16

You can install the cask, though. This will install the M1 version. Try `brew install --cask julia`
